blocks/to-do/to-do.js

```diff
@@ -79,23 +79,27 @@ export default function decorate(block) {
   toDoInputAndBtn.appendChild(input);
   block.appendChild(orderedList);
 
+  // Fetch data asynchronously
+  const data = await getListItems('to-do.js');
+
   // Call getListItems function to fetch data
-  getListItems().then((data) => {
-    // Loop through the data and add list items
-    data.forEach((item) => {
-      // Check if the "Status" is "true" before adding the item
-      if (item.Status === 'true') {
-        addListItem(null, item['to-do'], orderedList);
-      }
-    });
+  // Loop through the data and add list items
+  data.forEach((item) => {
+    // Check if the todo "completed" value is true which comes back as a string
+    if (item.completed === 'true') {
+      addListItem(item['to-do'], orderedList);
+    } else {
+      addListItem(item['to-do'], orderedList);
```

Comment:
I guess this is still in progress, because there is not really any difference between the two options (if/else)